### PR TITLE
POST-DEMO Build out Customer Info panel display [delivers #157604360]

### PR DESCRIPTION
**Pull request by jim**
## Description

This PR adds the display view for the Customer Info panel.

## Reviewer Notes

* Most of the fields in this panel involve multiple values being displayed together, which makes trying to extract field titles from the Swagger definitions tricky. I also noticed that many of the fields here don't have `title`s defined in `internal.yaml`.

## Code Review Verification Steps

* [ ] All tests pass.
* [ ] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [ ] This works in IE.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/157604360) for this change

## Screenshots

IE11:
screen shot 2018-05-15 at 7 52 06 pm

### CODE REVIEW

**Comment by breanneboland** [comment]
Once I caught on that I needed to make a move - something with those details beyond our test data - this worked fine. Before I did that, though, I was trying to click on one of the Spacemen lines and got a dismaying error: `TypeError: Cannot read property 'street_address_1' of undefined` 💥 

Considering things, I'm not sure if it makes sense to alter how we handle that data to account for gaps or just note it in our thoughts for near-future data generation. I think this looks good, but I'm sitting on my official approval until someone with greater context can address that - because otherwise, it's merging something that will break with most of the data we're populating the queue table with.

File: src/scenes/Office/CustomerInfoPanel.jsx [hunk 1]
```diff
@@ -143,10 +217,10 @@ function mapStateToProps(state) {
     initialValues: {},
```
Comment (kilbergr) [inline src/scenes/Office/CustomerInfoPanel.jsx]: I don't think you need this ^

File: src/scenes/Office/CustomerInfoPanel.jsx [hunk 2]
```diff
@@ -8,14 +8,88 @@ import editablePanel from './editablePanel';
...
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import faPhone from '@fortawesome/fontawesome-free-solid/faPhone';
+import faComments from '@fortawesome/fontawesome-free-solid/faComments';
+import faEmail from '@fortawesome/fontawesome-free-solid/faEnvelope';
+
 const CustomerInfoDisplay = props => {
-  // const fieldProps = pick(props, ['schema', 'values']);
+  const fieldProps = pick(props, ['schema', 'values']);
+  const values = props.values;
+  const name = `${values.last_name}, ${values.first_name}`;
```
Comment (kilbergr) [inline src/scenes/Office/CustomerInfoPanel.jsx]: I think there needs to be greater protection on how these are loading. For example, if I don't have a first name, I get `Spaceman, `. 
Do we think it's impossible that any of this data will be missing? Or do we need to handle it throughout?

Comment (kilbergr) [reply src/scenes/Office/CustomerInfoPanel.jsx]: I'm doing as follows:
```
    <PanelField title="Current Duty Station">
          {values.current_duty_station && `${values.current_duty_station.name}`}
        </PanelField>
```

**Comment by breanneboland** [comment]
Works beautifully for me now, Spacemen or no.

**Pull request merged**